package.json

```diff
@@ -43,7 +43,7 @@
     "sinon-chai": "^3.3.0"
   },
   "engines": {
-    "node": ">= 8"
+    "node": ">= 12"
```

Comment:
```suggestion
    "node": "12.* || 14.* || >= 16"
```

Node 13 and 15 are unsupported already

Reply:
We should have a semver string like this handy every time we have to roll through a bunch of projects and update. Maybe even a codemod?